### DISCUSSION TITLE
[VTS] Size and Liveliness as they currently are calculated are inversely proportional to one another

### DISCUSSION THREAD

**Discussion by onmax** [nimiq/developer-center#22] [General]
_Originally posted by @nibhar in https://github.com/nimiq/developer-center/issues/21#issuecomment-2141638059_

As a higher level comment after an initial quick read it seems to me that `Size` and `Liveliness` as they currently are calculated are inversely proportional to one another. That is probably not ideal as they sort of measure the same thing (bigger size gets you selected more often), but the score adjustments oppose each other.

An option would be to move Liveliness from purely being selected as the criteria, to not being selected due to a different reason than randomness (being jailed, being deactivated, etc). That is also implied by the paragraph leading up to it. This too must be done carefully as `Reliability` is also already a factor and we likely do not want to measure that one again either.

Another would be to scratch `Liveliness` if it does not add something new and its function can be absorbed into either `Reliability` or `Size`

**Comment by sisou** [comment]
Maybe this inverseness is on purpose. A good pool must not become too big to become a risk to the network, but also not too small to not get elected regularily. Weighting risk vs consistency. I think this is actually good. 

I would consider an adjustment to the `size` curve, such that is doesn't immediately start going down from the perfect `1` score, but only starts going down at 5% stake, for example.
